stdlib/public/runtime/SwiftObject.h

```diff
@@ -72,13 +72,13 @@ SWIFT_RUNTIME_EXPORT @interface SwiftObject<NSObject> {
 - (instancetype)autorelease;
 - (NSUInteger)retainCount;
 
-- (NSString *)description;
-- (NSString *)debugDescription;
+- (id)description;
+- (id)debugDescription;
```

Comment:
Maybe you can put the stringyness in a comment? `id /* NSString */` or something.

Reply:
or a typealias, good thought